tests/test_lizzy.py

```diff
@@ -87,106 +101,55 @@ def test_traffic(monkeypatch):
 
     header = make_header('7E5770K3N')
     mock_patch.assert_called_once_with('https://lizzy.example/api/stacks/574CC',
-                                       headers=header, data='{"new_traffic": 42}',
+                                       headers=header,
+                                       data=None,
+                                       json={"new_traffic": 42},
                                        verify=False)
 
 
-def test_new_stack(monkeypatch):
-    test_dir = os.path.dirname(__file__)
-    yaml_path = os.path.join(test_dir, 'test_config.yaml')  # we can use any file for this test
-    with open(yaml_path) as yaml_file:
-        senza_yaml = yaml_file.read()
-
+@pytest.mark.parametrize(
+    "version, parameters, region, disable_rollback, dry_run, force, tags, keep_stacks, new_traffic",
+    [
+        ("new_version", ['10'], None, True, False, False, [], 2, 42),
+        ("another_version", ['10'], "eu-central-1", False, True, False, [], 2,
+         42),
+        ("yet_another_version", [], None, False, False, True, [], 42, 7),
+        ("43", ['abc', 'def'], None, True, False, True, ['tag1=value1'], 42, 7),
+        ("newer_version", [], None, True, True, False, [], 2, 42),
+    ])
```

Comment:
This approach on testing is very bad in my opinion. Just expressing my concern, we do not need to change that if you like it.

In my opinion it is very bad because is totally not clear what it is testing, test is difficult to understand, and difficult to maintain with all those value to argument mapping etc.